assets/src/components/checkboxControl/style.scss

```diff
@@ -30,4 +30,11 @@
 		margin-left: 2em;
 		font-size: 16px;
 	}
+
+	.components-base-control__help {
+		margin-left: 58px;
+		font-style: normal;
+		margin-top: 0;
+		color: $muriel-gray-600;
+	}
```

Comment:
The help prop was unstyled for our checkboxes, causing it to go under the checkbox which looked ugly. I've added styling for it to make it look like our designs.